game/characters/wizard.py

```diff
@@ -35,7 +45,7 @@ def attack(self, surface, target):
                     self.projectile_sound.play()
                     self.projectiles.append(
                         Projectile(self.rect.centerx - (2 * self.rect.width * self.flip), self.rect.y, self.rect.width,
-                                self.rect.height // 2, damage, knockback, self, 10 - (20 * self.flip)))
+                                self.rect.height // 2, damage, knockback, self, 10 - (20 * self.flip), pulse))
```

Comment:
same here, we should probably just load the images once and pass the scaled version to the projectile constructor